README-spec.md

```diff
@@ -3,6 +3,8 @@
 Note that we expect no more than 10 simultaneous users. This app
 will never require scalability, so don't optimize for scale.
 
+We should use HTMX not JS.
```

Comment:
HTMX is a new web development approach that combines HTML, CSS, and JavaScript with server-side technologies to create fast and dynamic web applications. Choosing HTMX over JS could be a wise decision.